stages/org.osbuild.oscap.remediation

```diff
@@ -11,6 +12,27 @@ XCCDF_RESULTS = "oscap_eval_xccdf_results.xml"
 REMEDIATION_SCRIPT = "oscap_remediation.bash"
 
 
+def get_tailoring_id(tailoring_file):
```

Comment:
I took a step back. I think there is a much simpler approach.

> However, data passed in the command line options takes precedence over JSON contents, including the 
BASE_PROFILE_ID argument [1]

The above comes from `autotailor(8)`. We can add an additional arg to the json tailoring that takes in `new_profile_id` with the suffix of `_osbuild_tailoring`. And then we won't have any of these issues. And we can drop this commit.

Since https://github.com/osbuild/osbuild/pull/1777 is ready, I can do this as a follow up in this PR instead.